bin/fuel-convert

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+import argparse
+import importlib
+import pkgutil
+
+from fuel import converters
+
+
+if __name__ == "__main__":
+    dataset_options = [
+        m[1] for m in pkgutil.iter_modules(converters.__path__) if not m[2]]
+    parser = argparse.ArgumentParser()
+    parser.add_argument("dataset", help="what dataset to convert",
+                        choices=dataset_options)
+    parser.add_argument("-d", "--directory", help="directory in which input " +
```

Comment:
Right, but the only reason we put files in separate directories right now is because datasets could consist of a multitude of files. If we store everything in a single HDF5 file, I think the dataset should expect to read `$FUEL_DATA_PATH/mnist.hdf5`, not `$FUEL_DATA_PATH/mnist/mnist.hdf5`, which is a bit redundant.

The workflow is then simply:
```bash
cd $FUEL_DATA_PATH
curl -O [mnist_files]
fuel-convert mnist
```